docs/spec.md

```diff
@@ -2702,19 +2713,25 @@ for `fft_type = RFFT`. For example, for `L = 3`:
 * `result2[i0, ..., :, iR-1] = ifft(result1[i0, ..., :, iR-1])`.
 * `result[i0, ..., :] = irfft(result2[i0, ..., :])`.
 
+For quantized types, performs:
+
+* `fft(dequantize(operand), fft_type, fft_length)` if `fft_type = RFFT`.
+* `quantize(fft((operand, fft_type, fft_length), type(result)))` if `fft_type
+   = IRFFT`.
```

Comment:
Hmm I don't know. This looks sufficiently weird that I think we should just decline supporting quantized types in FftOp, similarly to the overall guideline that quantized types don't compose with complex types.

Reply:
I see you point. Let us not fuse de-quantization or quantization in this op. Updated the PR description with the revised proposal.